src/allotropy/parsers/utils/values.py

```diff
@@ -183,7 +191,7 @@ def try_bool_from_series_or_none(
 ) -> Optional[bool]:
     try:
         value = data.get(key)
-        return None if value is None else bool(value)
+        return None if value is None else str(value).lower() == "true"
```

Comment:
Ah, I should have caught this originally.  Can you add a function like this:

```
def str_to_bool(v):
  return v.lower() in ("yes", "true", "t", "1")
``` 

And call that instead?

```
return None if value is None else str_to_bool(value)
```